web/Controllers/AbstractController.cs

```diff
@@ -2,6 +2,7 @@
 
 using BricksAndHearts.Auth;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Azure;

```

Comment:
Don't think this is used :)

Reply:
I distinctly remember deleting that, it must have crept back in a rebase.